src/rinterface.h

```diff
@@ -27,3 +27,10 @@
 #define R_IGRAPH_VERSION_VAR ".__igraph_version__."
 
 SEXP R_igraph_add_env(SEXP graph);
+
+#define IGRAPH_R_CHECK(func) \
+    do { \
+        if (0) {igraph_error_type_t __c = func; } \
```

Comment:
What do we need the `if (0)` for? Can you please add a comment to explain?

Reply:
Remove this from here. In the C core this looks different and is essentially for a static compiler check, will explain when I get back next week.